docs/installation/mac.md

```diff
@@ -10,31 +10,25 @@ parent = "smn_engine"
 
 # Mac OS X
 
-You can install Docker using Boot2Docker to run `docker` commands at your command-line.
-Choose this installation if you are familiar with the command-line or plan to
-contribute to the Docker project on GitHub.
+You install Docker using Docker Toolbox. Docker Toolbox includes the following Docker tools:
 
-[<img src="/installation/images/kitematic.png" alt="Download Kitematic"
-style="float:right;">](https://kitematic.com/download)
-
-Alternatively, you may want to try <a id="inlinelink" href="https://kitematic.com/"
-target="_blank">Kitematic</a>, an application that lets you set up Docker and
-run containers using a graphical user interface (GUI).
-
-## Command-line Docker with Boot2Docker
+* Docker Machine binary for running the `docker-machine` binary
+* Docker Engine binary for running the `docker` binary
+* Docker Compose binary for running the `docker-compose` binary
+* Kitematic, the Docker GUI
+* a shell preconfigured for a Docker command-line environment
+* Oracle VM VirtualBox 
 
 Because the Docker daemon uses Linux-specific kernel features, you can't run
-Docker natively in OS X. Instead, you must install the Boot2Docker application.
-The application includes a VirtualBox Virtual Machine (VM), Docker itself, and the
-Boot2Docker management tool.
+Docker natively in OS X. Instead, you must use `docker-machine` to create and attach to a Docker VM on your machine. This VM hosts Docker for you on your Mac.
 
-The Boot2Docker management tool is a lightweight Linux virtual machine made
-specifically to run the Docker daemon on Mac OS X. The VirtualBox VM runs
-completely from RAM, is a small ~24MB download, and boots in approximately 5s.
+The Docker VM is lightweight Linux virtual machine made specifically to run the
+Docker daemon on Mac OS X. The VirtualBox VM runs completely from RAM, is a
+small ~24MB download, and boots in approximately 5s.
 
 **Requirements**
 
-Your Mac must be running OS X 10.6 "Snow Leopard" or newer to run Boot2Docker.
+Your Mac must be running OS X 10.6 "Snow Leopard" or newer to run Docker.
```

Comment:
Is this still true? What version of VirtualBox do we install, Because support for OS X 10.6 and 10.7 was removed in VirtualBox 5, and only OS X 10.8 "Mountain Lion" and above is now supported; 
https://www.virtualbox.org/manual/ch01.html#hostossupport
https://www.virtualbox.org/manual/ch01.html#ftn.idp91671472